paper.tex

```diff
@@ -151,36 +146,38 @@ \section*{Introduction}
 principles, but share some common properties. 
 Firstly, they are all 
 hueristic and approximation driven, favouring computational
-scalability over statistical rigor.
+scalability over a basis in a statistical model.
 Secondly, they all produce a single
 ARG as output, inferring a deterministic point estimate with no means 
 of quantifying uncertainty. Thirdly (and most importantly for the 
-purposes of this paper) the ARGs they produce lack substantial
-detail about ancestral recombination events: the estimates are
-rough sketches, hopefully capturing large-scale properties, but lacking
-in the finer detail of outputs of ARGweaver or SINGER. 
+purposes of this paper) the ARGs they produce 
+lack some of the information present in ARGweaver or SINGER. 
```

Comment:
I think that the general language around "rough sketches" is imprecise and also too pessimistic. One might even take the opposite view that SINGER is "overly precise" in that it gives you estimates of things you can't possibly know. (This is not a correct criticism because it gives you samples from the posterior, but still.)

Reply:
So I've tried to say more precisely in waht follows what we're talking about here.

Another - I think more accurate - way of viewing the distinction is that the different methods treat uncertainty differently: tsinfer reports it "structurally" (by omitting things it doesn't have a good guess at), and SINGER by taking posterior samples.

Reply:
I like it.